src/SavingsDai.sol

```diff
@@ -234,6 +235,9 @@ contract SavingsDai {
         }
 
         emit Deposit(msg.sender, receiver, assets, shares);
+        if(referralCode > 0) {
```

Comment:
Missing space after the `if`.
Also, consider doing the event emit in the same line.

Reply:
Done